build/source/driver/multi_driver.f90

```diff
@@ -189,7 +190,7 @@ program multi_driver
 type(gru_hru_double)             :: dparStruct                 ! x%gru(:)%hru(:)%var(:)     -- default model parameters
 ! define indices
 integer(i4b)                     :: iStruct                    ! loop through data structures
-integer(i4b)                     :: iGRU,jGRU                  ! index of grouped response unit
+integer(i4b)                     :: iGRU,jGRU,kGRU             ! index of grouped response unit
```

Comment:
Can i,j,k be given more meaningful names?

Reply:
or at least some comment here on what they are used for?  I tend to think of i,j,k as being nested loops which isn't really what you are doing (from what I can see? )

Reply:
They are used to index different elements of the GRU structure (as stated). This is none in different ways in different parts of the code, so adding more meaningful names is not possible without adding more variables. Will retain as is.